hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/upgrade/OMLayoutFeature.java

```diff
@@ -40,7 +40,9 @@ public enum OMLayoutFeature implements LayoutFeature {
 
   MULTITENANCY_SCHEMA(3, "Multi-Tenancy Schema"),
 
-  HSYNC(4, "Support hsync");
+  HSYNC(4, "Support hsync"),
+
+  SNAPSHOT_SUPPORT(5, "Ozone version supporting snapshot");
```

Comment:
Call this `FILESYSTEM_SNAPSHOT` instead? `_SUPPORT` is not necessary IMO. And we want to differentiate it from Ratis snapshots

```suggestion
  FILESYSTEM_SNAPSHOT(5, "Ozone version supporting snapshot");
```

Reply:
yup make sense! thanks siyao!

Reply:
updated